### ISSUE TITLE
afv command issues

### ISSUE THREAD

**Issue by radare** [radare/radare2#5459]
```
[0x100001174]> afv
See afv?, afvb?, afvr? and afvs?
[0x100001174]> afv?
|Usage: afv[rbsa]
| afvr?  manipulate register based arguments
| afvb?  manipulate bp based arguments/ vars
| afvs?  manipulate sp based arguments/ vars
| afva   analyze function arguments/vars
```

afv - list local variables ..like `pd 1 ~var` 
afv* - list local vars in commands
afvn - this command doesnt exist anymore and was suposed to be used to rename vars.. so this breaks the Vvn menu

The r_anal_var api should be probably moved to be under the fcn_ namespace but it makes thing much longer.

**Comment by radare** [comment]
Also the Vcdn is broken to rename vars

**Comment by radare** [comment]
oh i see why you killed afvn.. but that was useful, because names must be UNIQUE, a function cant have a var with the same name of an arg or so. so it was kind handy

**Comment by oddcoder** [comment]
then their is no problem right ?

**Comment by radare** [comment]
Hold on i have a partial fix for this in localhost. I had to go away but ill finish and push later at night

**Comment by radare** [comment]
Fixed some here 3e5f6368691365bcfd632cf41dc3e01e6d4dec0e
